src/recombinations.jl

```diff
@@ -481,6 +481,21 @@ function POS(v1::T, v2::T; rng::AbstractRNG=Random.GLOBAL_RNG) where {T <: Abstr
     return c1,c2
 end
 
+# Combinatorial crossovers
+# ----------------------
+
+"""Subset crossover"""
+function SXO(v1::T, v2::T; rng::AbstractRNG=Random.GLOBAL_RNG) where {T <: AbstractVector}
```

Comment:
It's a binary crossover, so it's better to constrain input type to `AbrstractVector{Bool}`.

Reply:
The latest commit implements your first suggestion.

I don't think I quite grasp the second one though. For my use case, the only valid value for `K` is the current number of `true` entries in each parent (which must be equal). Selecting more or less than that would be an error. Can you describe an example of how the use case you are thinking of?

Reply:
I thought about it and there's a different way to generalize SXO (btw, not married to this name, just tried to follow what seems the norm in the field). We could expand it to `v <: AbstractVector{<:Int}`s by considering that `v[i]=c` indicates that the `i`-th element is assigned to the `c`-th class, for `c in 1:C` and `C>=2`. The `BitVectors` case would correspond to `C=2`.